internal/service/admin/admin.service.go

```diff
@@ -16,13 +16,15 @@ import (
 	"github.com/gin-gonic/gin"
 	"github.com/nicksnyder/go-i18n/v2/i18n"
 	"go.uber.org/zap"
+	"gorm.io/gorm"
 )
 
 type IAdminService interface {
 	CreateStudentAccount(ctx *gin.Context, input *admin_dto.AdminCreateStudentAccountInput) (int, error)
 	CreateTeacherAccount(ctx *gin.Context, input *admin_dto.AdminCreateTeacherAccountInput) (int, error)
 	UploadFileStudentData(ctx *gin.Context, file *multipart.FileHeader) (int, *import_dto.ImportOutput)
 	UploadFileTeacherData(ctx *gin.Context, file *multipart.FileHeader) (int, *import_dto.ImportOutput)
+	UpdateAccount(ctx *gin.Context, userID int64, input *admin_dto.UpdateAccountInput) (int, error)
```

Comment:
Pass the userID into UpdateAccountInput, the tag will be swagger ignore and add the value to the input in the controller instead pass 3 params like this